### ISSUE TITLE
Failed to get the sub and auto-sub.

### ISSUE THREAD

**Issue by tuanbs** [Hexer10/youtube_explode_dart#146]
I cannot get the available subtitle and auto-generated subs recently. Got the following error:
```bash
FatalFailureException: Failed to perform an HTTP request to YouTube due to a fatal failure.
      In most cases, this error indicates that YouTube most likely changed something, which broke the library.
      If this issue persists, please report it on the project's GitHub page.
      Request: GET https://youtube.com/get_video_info?video_id=IFhns9MOUt8&el=embedded&eurl=https%3A%2F%2Fyoutube.googleapis.com%2Fv%2FIFhns9MOUt8&hl=en&html5=1&c=TVHTML5&cver=6.20180913
      Response: (404)
```

It worked few days ago. Please help. Thanks in advance.

**Comment by Hexer10** [comment]
This should be fixed in the latest commit: v`1.10.1`

**Comment by tuanbs** [comment]
@Hexer10 Thank you very much sir. I've just tried the latest one and it works like a charm. The interesting thing I found in this package is that there are some videos I cannot download the auto-translated sub using the popular `YouTube-dl`, but I can get it using this package.
For example:
```bash
youtube-dl --write-auto-sub --sub-lang=vi --skip-download "https://www.youtube.com/watch?v=uOuwOc5l1AA"
```
The above cmd doesn't generate anything, but this package can generate it.
Anyway, thanks for this awesome package. Cheers.
